docker/test/integration/minifi/validators/NoFileOutPutValidator.py

```diff
@@ -12,15 +11,13 @@ class NoFileOutPutValidator(FileOutputValidator):
     def __init__(self):
         self.valid = False
 
-    def validate(self, dir=''):
-
-        if self.valid:
-            return True
-
-        full_dir = self.output_dir + dir
+    def validate(self):
+        self.valid = False
+        full_dir = os.path.join(self.output_dir)
         logging.info("Output folder: %s", full_dir)
-        listing = listdir(full_dir)
 
-        self.valid = not bool(listing)
+        if not os.path.isdir(full_dir):
+            return self.valid
 
+        self.valid = (0 == self.get_num_files(full_dir))
         return self.valid
```

Comment:
this could be written in a shorter way:
```suggestion
        return os.path.isdir(full_dir) and 0 == self.get_num_files(full_dir)
```
and then `self.valid` can be removed

Reply:
Update in cdb014f306f614bfad5e6120f089ab821c904677